pkg/plugins/autodiscovery/cargo/dependencyManifest.go

```diff
@@ -0,0 +1,59 @@
+package cargo
+
+const (
+	// dependencyManifest is the Go template used to generate
+	// the Helm chart manifests specific for Helm dependencies
+	dependencyManifest string = `name: '{{ .ManifestName }}'
+sources:
+  {{ .SourceID }}:
+    name: '{{ .SourceName }}'
+    kind: 'cargopackage'
+    spec:
+      package: '{{ .DependencyName }}'
+      versionFilter:
+        kind: '{{ .SourceVersionFilterKind }}'
+        pattern: '{{ .SourceVersionFilterPattern }}'
+{{- if .WithRegistry }}
+      registry:
+        url: '{{ .RegistryURL }}'
+        rootdir: '{{ .RegistryRootDir }}'
+        auth:
+          token: '{{ .RegistryAuthToken }}'
+          headerFormat : '{{ .RegistryHeaderFormat }}'		
+{{- if .RegistrySCMID }}
+    scmid: '{{ .RegistrySCMID }}'
+{{- end }}
+{{- end }}
+  {{ .ExistingSourceID }}:
+    name: '{{ .ExistingSourceName }}'
+    kind: 'toml'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      file: '{{ .File }}'
+      Key: '{{ .ExistingSourceKey }}'
+conditions:
+  {{ .ConditionID }}:
+    name: 'Ensure Cargo chart named "{{ .DependencyName }}" is specified'
+    kind: 'toml'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      file: '{{ .File }}'
+      Query: '{{ .ConditionQuery }}'
```

Comment:
```suggestion
      query: '{{ .ConditionQuery }}'
```

keys should all be lowercase

Reply:
while the uppercase key should work. The jsonschema generated would display would consider it as an error